From 591632993b969d79dac03ad661c2b90c322ba91d Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 12 Sep 2017 17:31:22 +0800
Subject: [PATCH 4520/5242] MLK-19413-9 gpu: imx: dpu: Kconfig: Select IMX8_PC

commit  e8c4fff55fb4c5a6d144933ae25b9dd1ddfca723 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch selects IMX8_PC when DPU core is enabled.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/imx/dpu/Kconfig b/drivers/gpu/imx/dpu/Kconfig
index 3c680bb..7ccba74 100644
--- a/drivers/gpu/imx/dpu/Kconfig
+++ b/drivers/gpu/imx/dpu/Kconfig
@@ -3,6 +3,7 @@ config IMX_DPU_CORE
 	depends on ARCH_FSL_IMX8QM || ARCH_FSL_IMX8QXP
 	select RESET_CONTROLLER
 	select GENERIC_IRQ_CHIP
+	select IMX8_PC
 	select IMX8_PRG
 	select IMX8_DPRC
 	help
-- 
1.7.9.5

